pkg/logs/config/config.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package config
+
+import (
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util"
+
+	log "github.com/cihub/seelog"
+	"github.com/spf13/viper"
+)
+
+// LogsAgent is the global configuration object
```

Comment:
Dubious naming for the global config object ?

Reply:
Agreed but I won't fix this now, it's just code that I moved around. My plan is to get ride of this global variable in another PR, I can add a note though :)